src/main/java/org/folio/inventory/validation/ItemsValidator.java

```diff
@@ -1,29 +1,29 @@
 package org.folio.inventory.validation;
 
-import static io.vertx.core.Future.failedFuture;
-import static io.vertx.core.Future.succeededFuture;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.inventory.support.CompletableFutures.failedFuture;
 
 import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
 
 import org.folio.inventory.domain.items.Item;
 import org.folio.inventory.domain.items.ItemStatusName;
-import org.folio.inventory.support.http.server.ValidationError;
 import org.folio.inventory.exceptions.NotFoundException;
 import org.folio.inventory.exceptions.UnprocessableEntityException;
-
-import io.vertx.core.Future;
+import org.folio.inventory.support.http.server.ValidationError;
 
 public final class ItemsValidator {
 
```

Comment:
Personally, I wouldn't bother with this blank line, I don't think it adds to the readability